test/test_core.py

```diff
@@ -8498,7 +8507,10 @@ def break_wasm(name):
         return False
       create_file('wat.wat', wat)
       shutil.move(name, name + '.orig')
-      self.run_process([Path(building.get_binaryen_bin(), 'wasm-as'), 'wat.wat', '-o', name, '-g'])
+      self.run_process([
+        Path(building.get_binaryen_bin(), 'wasm-as'),
+        'wat.wat', '-o', name, '-g',
+        '--enable-memory64' if self.get_setting('MEMORY64') else '--disable-memory64'])
```

Comment:
I think we can just pass `--alll-features` here and avoid the conditional (maybe keep it all on the single line to keep the diff small).